conf/app.routes

```diff
@@ -3,6 +3,7 @@
 POST        /v2/declaration                                  uk.gov.hmrc.exports.controllers.DeclarationController.post()
 GET         /v2/declaration                                  uk.gov.hmrc.exports.controllers.DeclarationController.get()
 GET         /v2/declaration/:id                              uk.gov.hmrc.exports.controllers.DeclarationController.getByID(id: String)
+DELETE      /v2/declaration/:id                              uk.gov.hmrc.exports.controllers.DeclarationController.deleteByID(id: String)
```

Comment:
I know that it is a bit late now, but would it not be more readable if we use more explanatory method names? `post` or `get` are not very descriptive, since they only tell about request types.